config/reasoner/op2dl/main/persoon_https_to_http.n3

```diff
@@ -0,0 +1,534 @@
+PREFIX : <#>
```

Comment:
didn't expect this one?

Reply:
It's a relative uri. I used this namespace for the backward arrow rules (i.e. date -> dateTime conversions) which are defined inside the mapping file.